### ISSUE TITLE
Could not find the SWI-Prolog library in this platform.

### ISSUE THREAD

**Issue by jiacheng-ye** [yuce/pyswip#148]
I've installed `swipl` with `snap` refering https://www.swi-prolog.org/build/snap.html.

But when I import `pyswip` in python, it raises `Could not find the SWI-Prolog library in this platform. If you are sure it is installed, please open an issue.`.

Also, I found after I run `swipl --dump-runtime-variables`, it shows:
```
PLBASE="/snap/swi-prolog/68/usr/lib/swipl";
PLARCH="x86_64-linux";
PLBITS="64";
PLVERSION="90003";
PLSOEXT="so";
PLSOPATH="LD_LIBRARY_PATH";
PLLIBDIR="/snap/swi-prolog/68/usr/lib/swipl/lib/x86_64-linux";
PLLIB="-lswipl";
PLLIBSWIPL="/snap/swi-prolog/68/usr/lib/libswipl.so.9";
PLSHARED="yes";
PLTHREADS="yes";
```
But I do not find `libswipl.so`:
<img width="921" alt="image" src="https://user-images.githubusercontent.com/30772464/208431168-bb1fcca4-8ed7-48a0-8485-0979b3c46007.png">

Any helps and thanks in advance.

**Comment by jiacheng-ye** [comment]
Btw, this is the system info:
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"
```

**Comment by VorpalBlade** [comment]
What is the version of swipl? PySwip is currently incompatible with SWI-Prolog 9.x.

**Comment by LyzardKing** [comment]
If somebody else has this issue, a while back I opened a PR that addresses this(#106). It's a simple fix you can make to the version of pyswip you have running

**Comment by yuce** [comment]
The recently released PySwip v0.3.0 should solve this issue when SWI-Prolog is installed using the recommended ways mentioned at https://pyswip.org/get-started.html
